src/test/java/com/dabsquared/gitlabjenkins/webhook/build/PushBuildActionTest.java

```diff
@@ -102,10 +103,13 @@ private String getJson(String name) throws IOException {
     public void scmSourceOnUpdateExecuted() {
         GitSCMSource source = new GitSCMSource("http://test");
         SCMSourceOwner item = mock(SCMSourceOwner.class);
+        ACL acl = mock(ACL.class);
         when(item.getSCMSources()).thenReturn(Collections.singletonList(source));
+        when(item.getACL()).thenReturn(acl);
         Assert.assertThrows(
                 HttpResponses.HttpResponseException.class,
                 () -> new PushBuildAction(item, getJson("PushEvent.json"), null).execute(response));
+        item.onSCMSourceUpdated(source);
```

Comment:
'onSCMSourceUpdated(jenkins.scm.api.SCMSource)' is deprecated, implementations of [SCMSourceOwner](https://javadoc.jenkins.io/plugin/scm-api/jenkins/scm/api/SCMSourceOwner.html) would prefer the [SCMEventListener](https://javadoc.jenkins.io/plugin/scm-api/jenkins/scm/api/SCMEventListener.html) extension point which allows for more fine-grained response to events, so prefer delivering event notification through [SCMHeadEvent.fireNow(SCMHeadEvent)](https://javadoc.jenkins.io/plugin/scm-api/jenkins/scm/api/SCMHeadEvent.html#fireNow(jenkins.scm.api.SCMHeadEvent)), [SCMSourceEvent.fireNow(SCMSourceEvent)](https://javadoc.jenkins.io/plugin/scm-api/jenkins/scm/api/SCMSourceEvent.html#fireNow(jenkins.scm.api.SCMSourceEvent)) or [SCMNavigatorEvent.fireNow(SCMNavigatorEvent)](https://javadoc.jenkins.io/plugin/scm-api/jenkins/scm/api/SCMNavigatorEvent.html#fireNow(jenkins.scm.api.SCMNavigatorEvent)) as appropriate.